README.md

```diff
@@ -1,4 +1,4 @@
 # RAMS
 [![DOI](https://zenodo.org/badge/307815774.svg)](https://zenodo.org/badge/latestdoi/307815774)
 
-RAMS model source code. To see full documentation, go to the [van den Heever group website](https://vandenheever.atmos.colostate.edu/vdhpage/rams/rams_docs.php). 
```

Comment:
I don't see what was changed here? Why is it showing a diff?

Reply:
I also don't see the difference. Maybe it's as simple as a space missing or something?